run-tests.sh

```diff
@@ -72,7 +79,20 @@ set -ex
 
 node merge_kokoro_logs.js
 
-if [ "$FAILED" != "" ]
+if [ "$FAILED" = "true" ]
 then
   exit 1
+else
+  if [ "$OS" = "Linux" ]
```

Comment:
I realized we might even just remove this condition here, since gsutil was exactly the reason it wouldn't work on mac.

Reply:
I will leave it for now because I think it is semantically correct to publish coverage results only once per PR, even if the command were to start working on Mac in the future.